ivy/functional/frontends/torch/nn/functional/convolution_functions.py

```diff
@@ -78,7 +76,6 @@ def _conv_transpose(
     ret = ret[unpad_slice]
     return ret
 
```

Comment:
I don't see a new failure on this so I guess it's okay. But is it beneficial in any way? Does `conv_general_transpose` have fewer nodes then `conv_general_dilated`?

Reply:
It is just that conv general dilated does not use conv2dtranspose, so want to see if I map, ideally should be one to one mapping (torch.conv2dtranspose to tf.conv2dtranspose)